### DISCUSSION TITLE
How to extend another vitest.config.ts,  in monorepo?

### DISCUSSION THREAD

**Discussion by gyhyfj** [vitest-dev/vitest#3170] [Q&A] [answered]
It's amazing for supporting workspace, thank you all very much.
In my monorepo, I'd like to code a `vitest-config` project, using auto-import, and some common config in it's config file, 
then I can extend this config file in my every app project.
It just like how I deal with my tsconfig and eslint config,
Could it be possible?

**Comment by sheremet-va** [comment, marked as answer]
It is not possible to use `extends` field. But you can import the other config and merge it yourself: https://vitest.dev/guide/workspace.html#configuration

**Reply by gyhyfj** [reply]
Thank you very much
